diagnostic.md

```diff
@@ -9,6 +9,15 @@ Assume all code executes in strict mode.
 
 Please list the JavaScript primitives and give one example of each.
 
+ Number, String, Boolean, Undefined, Null.
+
+ Examples:
+ Number: 3
+ String: 'hello'
+ Boolean: true
+ Undefined: television
```

Comment:
I'll give you this one, but to be clear, `television` would only return `undefined` if it indeed has not been defined in the code. Otherwise, an examples of Undefined is just `undefined`.